src/components/checkboxFilter/index.tsx

```diff
@@ -53,16 +55,35 @@ function CheckboxFilter<ItemKey extends string>({
             key={`filter_${name}_${item.label}`}
             name={`filter_${name}_${item.label}`}
             label={
-              <chakra.span
-                w="full"
-                display="flex"
-                justifyContent="space-between"
-              >
-                <chakra.span>{item.label}</chakra.span>
-                <chakra.span>
-                  {addThousandSeparatorToNumber(item.facet)}
+              item.image ? (
+                <chakra.div
+                  w="full"
+                  display="flex"
+                  alignItems="center"
+                  justifyContent="space-between"
+                >
+                  <chakra.div width="119px" height="80px">
+                    {item.image}
+                  </chakra.div>
+                  <chakra.span display="flex" flexDirection="column">
+                    <chakra.span>{item.label}</chakra.span>
+                    <chakra.span color="gray.400">
+                      {addThousandSeparatorToNumber(item.facet)}
+                    </chakra.span>
+                  </chakra.span>
+                </chakra.div>
+              ) : (
+                <chakra.span
```

Comment:
This is almost the same as the variant with image. Could we extract to a component? Thanks!

Reply:
After removing the text color,, now it makes sense extracting the component 👍🏼

Reply:
After the adjustments the wrapper contains other different styles, in my point of view it doesn't make sense to extract it.